Stripe/UINavigationBar+Stripe_Theme.m

```diff
@@ -18,6 +19,14 @@ - (void)stp_setTheme:(STPTheme *)theme {
     [self stp_artificialHairlineView].backgroundColor = theme.tertiaryBackgroundColor;
     self.barTintColor = theme.secondaryBackgroundColor;
     self.tintColor = theme.accentColor;
+
+    if ([STPColorUtils colorIsBright:theme.secondaryBackgroundColor]) {
```

Comment:
@bdorfman just out of curiosity, how does `UIBarStyleBlack` play with the `barTintColor` and `tintColor` properties? Will it just make the bar, uh, black?